### DISCUSSION TITLE
snapshot different from storybook static page

### DISCUSSION THREAD

**Discussion by maxkarkowski** [percy/percy-storybook#517] [Support]
## The problem

i am implementing the percy/storybook package right now and found a difference between the snapshot percy took and the actual storybook static html file. 

this is the snapshot percy took
<img width="617" alt="percy snapshot" src="https://user-images.githubusercontent.com/622841/154036781-7a8b6927-718f-4cf7-973f-bb0f69aa28c7.png">

and this is the actual storybook
<img width="853" alt="storybook static" src="https://user-images.githubusercontent.com/622841/154036786-9b15f52f-c432-47a2-8ac1-8ffa27a905e3.png">

u see the asterix breaks in a new line in the snapshot

## Environment

- Node version: 14.18.2
"@percy/cli": "^1.0.0-beta.75",
"@percy/storybook": "^4.1.0",

**Comment by Robdel12** [comment]
Hey @maxkarkowski! In order to debug this we'll need to know the snapshot ID or build URL. It could be a legitimate thing that's happening in whatever browser that's capturing the screenshot or it could be something else. No way to know without looking at the build though.

**Reply by maxkarkowski** [reply]
Hey @Robdel12 , here is an url [LInk](https://percy.io/2a8b916a/test-storybook/builds/15743151/changed/888437487?browser=edge&browser_ids=18%2C19%2C20%2C21&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=1160&widths=320%2C576%2C768%2C1160%2C1440 )


I am new to percy so i am not quite sure where to find the snapshot ID

**Reply by Robdel12** [reply]
No worries, thanks for that! Looking at the captured DOM vs what is rendered, there's invalid DOM that's being corrected. `<p>` tags can't have [flow content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#flow_content) within them (like `<divs>`). Only [phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#phrasing_content) is allowed to be a child. 

```html
<p>
  inklusive
   <div>
     <span class="comp-footnote-modal">
       <button type="button" data-testid="footnote-button-1" class="comp-footnote-icon-button">
        <svg></svg>
      </button>
    </span>
  </div>
</p>
```

When this is re-rendered in Percy's infrastructure, the browser corrects any invalid DOM, which is breaking the layout here. https://docs.percy.io/docs/debugging-sdks#invalid-html
